docs/ref/models/querysets.txt

```diff
@@ -1750,6 +1750,43 @@ See the :doc:`/topics/db/sql` for more information.
   filtering. As such, it should generally be called from the ``Manager`` or
   from a fresh ``QuerySet`` instance.
 
+
+Operators that return new ``QuerySet``\s
+----------------------------------------
+
+AND (``&``)
+~~~~~~~~~~~
+
+Combines two ``QuerySet``\s by merging their filters, equivalent to
+the ``AND`` operator.
```

Comment:
Add "SQL" - "equivalent to the SQL ``AND`` operator". Otherwise it's a bit tautological to say the "and" operator is equivalent to the "and" operator